### PR TITLE
Adding tests for "Unicode and passing strings" part 13

### PR DESCRIPTION
**Pull request by gabrieldemarmiesse**

### CODE REVIEW

File: docs/examples/tutorial/string/for_char.pyx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cdef char* c_string = "Hello world"
+
+cdef char c
+for c in c_string[:11]:
+    if c == 'A':
+        print("Found the letter A")
```
Comment (scoder) [inline docs/examples/tutorial/string/for_char.pyx]: Shouldn't there be an 'A' somewhere in the example string? ;-)
